converter/converter/src/main/java/org/osgi/util/converter/ConvertingImpl.java

```diff
@@ -810,6 +813,32 @@ public Object invoke(Object proxy, Method method, Object[] args)
 					if (cls.isAnnotation()) {
 						val = method.getDefaultValue();
 					}
+					else if (method.isDefault())
+					{
+						double javaVersion = Double.parseDouble(
+							System.getProperty("java.class.version"));
+						double java8 = 52.0;
+						if (javaVersion > java8)
```

Comment:
So invoking default methods is different for Java 8 and Java 9+?

Reply:
Yes. I didn't found one way that works in 8 and 9+